iac-extensions/terraform/src/test/resources/checks/PublicNetworkAccessCheck/azure.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_batch_account" "production"  {} # Noncompliant {{Omitting public_network_access_enabled allows network access from the Internet. Make sure it is safe here.}}
+
+resource "azurerm_batch_account" "production"  {
+  public_network_access_enabled = true # Noncompliant {{Make sure allowing public network access is safe here.}}
```

Comment:
You could add a precise location, to show that the whole attribute is highlighted.